index.html

```diff
@@ -0,0 +1,111 @@
+
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <link type="text/css" rel="stylesheet" href="style.css" />
+  <title>Web Workbook</title>
+</head>
+
+<body background="https://www.worldofwallpaper.com/media/wysiwyg/DEC180-Fractal-Geometric-Marble-Wallpaper-Rose-Gold-EA4_1.jpg" >
+  
+        <h1>
+      <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Quicksand" />
+       Sky's Workbook</h1>
+       
+<div class="timeline">
+<div class="container left">
+  <div class="content">
+       <section class="week">
+    <h2>Week 1</h2>
+    <a href=./01week/helloworld/index.html>Hello World</a>
+    <br />
+    <a href="https://rawgit.com/skymendes/web-workbook/gh-pages/01week/resume/index.html">Resume</a>
```

Comment:
The link is incorrect here. You are linking it to the "live" page. All you have to do is find it in your web-workbook and like it that way. Since all of your projects are under the same folder (web-workbook)